packages/spec/README.md

```diff
@@ -4,49 +4,57 @@ A consortium of schema library authors have collaborated to craft a standard int
 
 ## The Interface
 
-The `StandardSchema` interface defines four properties that a schema library must implement to be compatible.
+The `StandardSchema` interface defines one properties that a schema library must implement to be compatible.
 
 ```ts
 /**
- * The Standard Schema v1 interface.
+ * The Standard Schema interface.
  */
-export interface StandardSchema<Input = unknown, Output = Input> {
+interface StandardSchema<Input = unknown, Output = Input> {
+  /**
+   * The Standard Schema properties.
+   */
+  readonly "~standard": StandardSchemaProps<Input, Output>;
+}
+
+/**
+ * The Standard Schema properties interface.
+ */
+interface StandardSchemaProps<Input = unknown, Output = Input> {
   /**
    * The version number of the standard.
    */
-  readonly "~standard": 1;
+  readonly version: 1;
   /**
    * The vendor name of the schema library.
    */
-  readonly "~vendor": string;
+  readonly vendor: string;
   /**
    * Validates unknown input values.
    */
-  readonly "~validate": StandardValidate<Output>;
+  readonly validate: (value: unknown) => StandardResult<Output> | Promise<StandardResult<Output>>;
   /**
    * Inferred types associated with the schema.
    */
-  readonly "~types"?: StandardTypes<Input, Output> | undefined;
+  readonly types?: StandardTypes<Input, Output> | undefined;
 }
 ```
 
-- `~standard` stores the version number and can be used to test whether an object is a Standard Schema. 
-
-- `~vendor` stores the name of the schema libarry. This can be useful for performing vendor-specific operations in special cases. 
-
-- `~validate` is a function that validates unknown input and returns the output of the schema if the input is valid or an array of issues otherwise. This can be discriminated by checking whether the `issues` property is `undefined`.
-
-- `~types` is used to associate type metadata with the schema. This property should be declared on the schema's type, but is not required to exist at runtime. Authors implementing the schema are encouraged to use TypeScript's `declare` keyword or other means of avoiding runtime overhead. `InferInput` and `InferOutput` can be used to extract their corresponding types.
```

Comment:
@ssalbdivad if I remember it correctly, the `declare` keyword is no longer necessary because the property is marked as options. That's why I removed that part.

Reply:
Should we change it to:

Authors implementing a schema using a class are encouraged to use TypeScript's `declare` keyword or other means of avoiding runtime overhead.

Reply:
Sure that works for me.

Reply:
Done